components/vfs/manager.cpp

```diff
@@ -57,6 +57,14 @@ namespace VFS
             (*it)->listResources(mIndex, mStrict ? &strict_normalize_char : &nonstrict_normalize_char);
     }
 
+    void Manager::rebuildIndex()
+    {
+        for (std::vector<Archive*>::const_iterator it = mArchives.begin(); it != mArchives.end(); ++it)
+            (*it)->reset();
+
+        buildIndex();
+    }
```

Comment:
It's a bit of a pain since the VFS is created in the Editor class and passed down to each document at creation, and the setVFS function of the DocumentManager class does not seem to have been implemented fully. I'll see what I can do though.